digits/dataset/images/classification/views.py

```diff
@@ -156,12 +157,14 @@ def from_files(job, form):
     """
     # labels
     if form.textfile_use_local_files.data:
-        job.labels_file = form.textfile_local_labels_file.data.strip()
+        labels_file_from = form.textfile_local_labels_file.data.strip()
```

Comment:
This change is already on master. You might want to rebase the `tensorflow` branch into master first - then add your patches. This will keep commit history cleaner.